hw/ip/pinmux/data/pinmux.hjson.tpl

```diff
@@ -113,6 +141,43 @@
       default: "${wkup_cnt_width}",
       local: "true"
     },
+    { name: "NUsbDevPads",
```

Comment:
yeah i had thought about that too.... just sort of ....ignored that possibility for now.

Reply:
I think this is ok for the time being. 
I can take a look at how we could make this more generic during the upcoming pinmux refactoring.

Reply:
yeah let me know what you think. Because i think as we add more "stuff", the dio's just kind of get randomly piled together, and there's no clear indication anywhere exactly how it's piled together.

Reply:
I believe the aggregation order is currently defined 1) by the order in which the IPs get listed as DIO modules in the [top hjson](https://github.com/lowRISC/opentitan/blob/9650146375157484be86c48f22a81dbfbe3c321d/hw/top_earlgrey/data/top_earlgrey.hjson#L801-L807), and then 2) by the order in which the DIOs are declared in the individual hjson files. 

This is one of the things I plan to rework a bit, such that IO to pin assignments can be done in a more robust / flexible fashion.